go.mod

```diff
@@ -262,6 +265,8 @@ require (
 	github.com/t-yuki/gocover-cobertura v0.0.0-20180217150009-aaee18c8195c // indirect
 	github.com/tdakkota/asciicheck v0.2.0 // indirect
 	github.com/tecbot/gorocksdb v0.0.0-20191217155057-f0fad39f321c // indirect
+	github.com/tendermint/tendermint v0.35.9 // indirect
```

Comment:
This is weird, we should not be pulling this dependency in.

Reply:
Yeah something's wrong here. We should _definitely_ not be seeing Tendermint or tm-db being pulled in here.